gptel-gemini.el

```diff
@@ -236,27 +236,53 @@ See generic implementation for full documentation."
                     (not (= (point) prev-pt)))
           (pcase (get-char-property (point) 'gptel)
             ('response
-             (push (list :role "model"
-                         :parts
-                         (list :text (buffer-substring-no-properties (point) prev-pt)))
-                   prompts))
+             (when-let* ((content (buffer-substring-no-properties (point) prev-pt)))
```

Comment:
The idea here is to avoid sending an empty response, right?  `(buffer-substring-no-properties (point) prev-pt)` will never be nil because `(point)` will never equal `prev-pt`, so this `when-let*` does nothing.  Please remove this check.

Reply:
e4dc8e7  Now it can avoid sending the empty message via trimming.